project/src/components/favorite-location/favorite-location.tsx

```diff
@@ -19,11 +19,14 @@ function FavoriteLocation({offers, cityName}: FavoriteLocationProps): JSX.Elemen
         </div>
       </div>
       <div className="favorites__places">
-        {offers.map((offer) => (
-          <OfferCard
-            key={offer.id}
-            offer={offer}
-          />))}
+        <OffersList
+          offers={offers}
+          className={'favorites__card'}
```

Comment:
Можно упростить запись:
```suggestion
          className="favorites__card"
```